src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml.cs

```diff
@@ -32,8 +32,23 @@ public InCanvasSearchControl()
             InitializeComponent();
             if (Application.Current != null)
             {
-                Application.Current.Deactivated += (s, args) => { OnRequestShowInCanvasSearch(ShowHideFlags.Hide); };
+                Application.Current.Deactivated += currentApplicationDeactivated;
```

Comment:
Not really a code review comment, but I think this type of change should be in the coding standards (i.e. no attaching lambda functions to event handlers, unless there is a specific reason).

Reply:
will add this to style guide.